cmd/panfs.go

```diff
@@ -1827,3 +1842,7 @@ func (fs *PANFSObjects) isConfigAgentObject(bucket, object string) bool {
 
 	return true
 }
+
+func (fs *PANFSObjects) getTempDir(bucketDir string) string {
+	return pathJoin(bucketDir, panfsS3TmpDir, strconv.FormatUint(atomic.AddUint64(&fs.currentTmpFolder, 1)%fs.tmpDirsCount, 10), fs.fsUUID)
```

Comment:
I'd recommend to swap fsUUID and slot number in path, so the node-specific folder will always go first, as:

```
return pathJoin(bucketDir, panfsS3TmpDir, fs.fsUUID, slotNum)
```

instead of

```
return pathJoin(bucketDir, panfsS3TmpDir, slotNum, fs.fsUUID)
```

I will also fill a ticket to make fs.fsUUID bound to node serial number instead of random UUID, so we can avoid infinite growing of these folders with minio restarts.

Reply:
Changed the order.
So, talking about fsUUID, which ID do you want me to take instead of random one? From env variable? Or maybe something from fsPath?

Reply:
Let's take in from environment variable, called PANFS_DATASERIAL. If this variable is not provided - we can generate random UUID (for backwards compatibility). I'd update the way how we initialize this fsUUID in NewPANFSObjectLayer.

Reply:
We also can rename this value to nodeDataSerial if its usage is limited only to PanFS backend (seems this should be the case).

Reply:
Changed to env variable with random default and renamed